vm/vm.go

```diff
@@ -379,12 +399,19 @@ func (vm *VM) checkActivity(ctx context.Context) {
 }
 
 func (vm *VM) markReady() {
+	// Wait for state syncing to complete
 	select {
 	case <-vm.stop:
 		return
 	case <-vm.stateSyncClient.done:
 	}
+
+	// We can't verify the block here because
```

Comment:
nit: comment that we begin verifying here but our verification is not fully sufficient